server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -312,7 +334,53 @@ void shouldRecomputeQuotas(GuiceJamesServer server) throws Exception {
     }
 
     @Test
-    void solveCassandraMappingInconsistencyShouldSolveNotingWhenNoInconsistencies() {
+    void shouldSolveMessagesInconsistency(GuiceJamesServer server) {
+        // schema version 6 or higher required to run solve message inconsistencies task
+        String upgradeTaskId = with().post(UPGRADE_TO_LATEST_VERSION)
+            .jsonPath()
+            .get("taskId");
+
+        with()
+            .get("/tasks/" + upgradeTaskId + "/await")
+        .then()
+            .body("status", is("completed"));
+
+        MailboxProbeImpl probe = server.getProbe(MailboxProbeImpl.class);
+        MailboxPath inbox = MailboxPath.inbox(BOB);
+        MailboxId mailboxId = probe.createMailbox(inbox);
+
+        TestingSessionProbe testingProbe = server.getProbe(TestingSessionProbe.class);
+        testingProbe.getTestingSession().registerScenario(fail()
+            .times(6) // Insertion in the DAO is retried 5 times before it fails
```

Comment:
Can we give `6` a better variable name to remove the comment?

Reply:
Propose a name then :-)

I am not sure we can condense all the information here in a variable name.

Reply:
How about DAO_INSERTION_MAX_RETRIES ?

Reply:
> How about DAO_INSERTION_MAX_RETRIES ?

Im ok with it.

Reply:
DAO_DENORMALIZATION_TOTAL_TRIES then